test/suites/network/multi-nic.robot

```diff
@@ -31,15 +34,57 @@ Verify MicroShift Runs On Both NICs
     # Wait for MicroShift API readiness and run verification
     Wait For MicroShift
     Verify Hello MicroShift LB
-    Verify Hello MicroShift NodePort    ${USHIFT_HOST_IP1}    ${USHIFT_HOST_IP2}
+    Verify Hello MicroShift NodePort    ${USHIFT_HOST_IP1}
+    Verify Hello MicroShift NodePort    ${USHIFT_HOST_IP2}
+    Check IP Present In External Certificate    ${USHIFT_HOST_IP1}
+    Check IP Present In External Certificate    ${USHIFT_HOST_IP2}
 
 Verify MicroShift Runs On First NIC
     [Documentation]    Verify MicroShift can run on the first NIC
-    Verify MicroShift On One NIC    ${USHIFT_HOST_IP1}    ${NIC2_NAME}    ${USHIFT_HOST_IP1}    ${EMPTY}
+    [Setup]    Save Default MicroShift Config
+
+    Setup MicroShift Config    ${USHIFT_HOST_IP1}
+
+    ${cur_pid}=    MicroShift Process ID
+
+    Login Switch To IP    ${USHIFT_HOST_IP1}
+    Nmcli Connection Control    down    ${NIC2_NAME}
+
+    Restart MicroShift
```

Comment:
Hmm, do we need to restart microshift in we disable one of the interfaces? Does sysconf controller only checks single (primary? how it was selected then) IP?

Reply:
The restart is required so that the new configuration is picked up again and certificates get renegerated. In this test case the host ip does not change because the interface is untouched.

Reply:
Yes yes, what I meant is the difference in behavior of primary and other interfaces, but I see in code it depends on default route (which says which interface is "primary") - and that's the only one we watch for changes